src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlConnectionStringBuilder.cs

```diff
@@ -132,7 +132,7 @@ private enum Keywords
         private bool _transparentNetworkIPResolution = DbConnectionStringDefaults.TransparentNetworkIPResolution;
         private string _networkLibrary = DbConnectionStringDefaults.NetworkLibrary;
 #else
-        internal const int DeprecatedKeywordsCount = 5;
```

Comment:
Can we get rid of this counter?

Reply:
I don't think so. It's used to mark keywords which are supported in .NET Framework but not in .NET Core. These are Connection Reset and Network Library. Connection Reset should be a fairly easy keyword to merge, but Network Library needs a little more thought - what connection method should we use if the Network Library keyword requests named pipes, but the Server keyword requests TCP?

Reply:
" if the Network Library keyword requests named pipes, but the Server keyword requests TCP"

Invalid connection string?